ConfigurationSystem/Client/VOMS2CSSynchronizer.py

```diff
@@ -268,6 +285,15 @@ def syncCSWithVOMS(self):
               if group not in noSyncVOMSGroups:
                 groupsWithRole.append(group)
           userDict['Groups'] = list(set(groupsWithRole + [defaultVOGroup]))
+
+          # Run the sync plugins for extra info and/or validations
+          if self.syncPlugin:
+            try:
+              self.syncPlugin.verifyAndUpdateUserInfo(newDiracName, userDict)
+            except ValueError as e:
```

Comment:
At the moment it's impossible to tell from the log that something went wrong. How about:
```suggestion
            except ValueError as e:
              self.log.exception("Error validating new user with nickname", newDiracName)
```

Reply:
I must admit I only went very briefly through this whole code, but I understand that adding the error in the `self.adminMsgs` was for further display. Am I wrong ?

Reply:
You are right. Bit the "in-place" log message would not harm neither.

Reply:
At least when I was testing this on the LHCb certification instance (in dry run mode) these errors were never printed which was very surprising. I got the impression adminMsgs was only sent via email but I could have misunderstood.

Reply:
Ok I added a message